### ISSUE TITLE
MPI build fails

### ISSUE THREAD

**Issue by montyvesselinov** [JuliaParallel/MPI.jl#254]
```
Julia Version 1.1.0
Commit 80516ca202 (2019-01-21 21:24 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) Silver 4116 CPU @ 2.10GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
Environment:
  JULIA_DEPOT_PATH = /home/vvv/.julia-ruml

 vvv@ruml.lanl.gov  09:48 ~/.julia-ruml/packages/MPI/b4xgC/deps [1343] mpiexec --version
mpiexec (OpenRTE) 1.10.7

Report bugs to http://www.open-mpi.org/community/help/
 vvv@ruml.lanl.gov  09:48 ~/.julia-ruml/packages/MPI/b4xgC/deps [1344] cmake -f CMakeLists.txt
-- Configuring done
-- Generating done
-- Build files have been written to: /home/vvv/.julia-ruml/packages/MPI/b4xgC/deps
 vvv@ruml.lanl.gov  09:48 ~/.julia-ruml/packages/MPI/b4xgC/deps [1344] make
[ 16%] Building Fortran object CMakeFiles/gen_constants.dir/gen_constants.f90.o
/home/vvv/.julia-ruml/packages/MPI/b4xgC/deps/gen_constants.f90:43.43:

  call output("MPI_NO_OP       ", MPI_NO_OP)
                                           1
Error: Symbol 'mpi_no_op' at (1) has no IMPLICIT type
make[2]: *** [CMakeFiles/gen_constants.dir/gen_constants.f90.o] Error 1
make[1]: *** [CMakeFiles/gen_constants.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by simonbyrne** [comment]
Same issue as #229. Looks like you're using an older MPI (pre 2.0): is there any way you can use a newer version?

**Comment by montyvesselinov** [comment]
Correct; I used 1.8; I can try ; what are the min requirements for openmpi and gcc; maybe cmake should check the versions.

**Comment by montyvesselinov** [comment]
installed openmpi 3.0 and i get the same error

**Comment by simonbyrne** [comment]
This should hopefully be fixed by #271. If not, please open a new issue.
